examples/hello-world/hello.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+import asyncio
+from typing import AsyncIterable
+
+# from yapapi.log import enable_default_logger, log_summary, log_event_repr  # noqa
+
+from yapapi import Golem, Task, WorkContext
+from yapapi.payload import vm
+
+
+async def worker(context: WorkContext, tasks: AsyncIterable[Task]):
+    async for task in tasks:
+        context.run("/bin/sh", "-c", "date")
```

Comment:
Good choice of a command! 

`context.run("/bin/date")` would be even simpler, but `context.run("/bin/sh", "-c", "date")` is more illustrative.